### PR TITLE
Add SPARQL queries for Bangla adjectives and prepositions

### LINKED ISSUE #134: Create needed queries for Bangla
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would create the needed Wikidata query service queries for Bangla, with specifically queries being made for nouns, verbs, prepositions, adjectives, and any other word type of interest. We can explore Wikidata a bit for this and decide what other word types would be good to include in the issue here :) Suggestions welcome!

A quick search is indicating that Bengali is the English name for Bangla, so work for this should go into [src/scribe_data/extract_transform/languages/Bengali](https://github.com/scribe-org/Scribe-Data/tree/main/src/scribe_data/extract_transform/languages/Bengali). Separate directories for word types should be added, and queries for each based on those for other langauges and based on available data can be made 😊

### Contribution

Happy to support the work on this! 🚀

### PR DESCRIPTION
**Pull request by mhmohona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This PR adds SPARQL queries for retrieving Bangla (Bengali) adjectives and prepositions from Wikidata.

### Related issue

Fixes #134

### CODE REVIEW

**Comment by mhmohona** [comment]
I am concerned about the preposition script as I am getting only 1 preposition from it. 

image

File: src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ?preposition
+  ?case
+
+WHERE {
+  ?lexeme a ontolex:LexicalEntry ;
+    dct:language wd:Q9610 ; # Bengali language
+    wikibase:lexicalCategory wd:Q4833830 ; # Prepositions
+    wikibase:lemma ?lemma .
+
+  # Optional selection of corresponding cases.
```
Comment (andrewtavis) [inline src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql:16]: Quick check here, @mhmohona: do Bangla prepositions have cases? So in German if you use the preposition `mit` meaning `with`, afterwards you need to use the `Dativ` case that will determine articles and other forms.

I just checked and it looks like Begali does have nominative and genitive cases, but wanted to check to make sure :) :)

Comment (andrewtavis) [reply src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql:16]: Aside from this, looks good! No stress on there not being any data. To be expected in some cases. When I first started Scribe French only had one fully conjugated verb 😊

**Review by andrewtavis — APPROVED** [review]
Approving with the assumption that the preposition query is correct, @mhmohona 😊 Let me know on the question of the cases, but looks good to me. I'm mostly just curious about how the language works 😇

Thanks so much for this!

File: src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql (hunk 1 above)
Comment (mhmohona) [reply src/scribe_data/extract_transform/languages/Bengali/prepositions/query_prepositions.sparql:16]: In this case Bangla is a easy language,it doesnt have case like German. 

The hard part of it is verb. For each person, each tense...almost for every thing it has different form of a particular verb.

**Comment by mhmohona** [comment]
Thank YOU for your support @andrewtavis! ^_^

**Comment by andrewtavis** [comment]
One note on this that came from the Wikidata Lexemes community is the Bangla there also are a lot of postpositions, @mhmohona :) Question is, do we make a different query for that or include both in one 🤔

**Pull request merged**